src/main/groovy/edu/oregonstate/mist/catalogapitest/db/CourseDAO.groovy

```diff
@@ -7,27 +7,48 @@ import org.skife.jdbi.v2.sqlobject.SqlQuery
 import org.skife.jdbi.v2.sqlobject.Bind
 import org.skife.jdbi.v2.sqlobject.customizers.RegisterMapper
 
+/**
+ * Course database access object class.
+ */
 @RegisterMapper(CourseMapper)
 public interface CourseDAO extends Closeable {
 
-    // Get all courses -------------------------------------------------------------------------------------------------
+    // /course ---------------------------------------------------------------------------------------------------------
```

Comment:
these line comments are potentially misleading because in theory the database class should not be concerned with endpoints defined in the resource class